### PR TITLE
Fix issues with post formats

### PR DESCRIPTION
**Pull request by dmytro-kovalov**
1. Fix the issue with displaying the markup under the post title. Replaced `get_the_content` with `the_excerpt` in the `content-video.php` template. Without the excerpt, the template will behave exactly like the `content-gallery.php`. User needs to add the excerpt for the post to add some description for the video. This does not affect the single post template, only the post preview (on pages like blog posts/archive/etc).

2. Fix issues with missing embeds in post previews. They are just not displayed. Modern embeds use blocks, so it requires some additional fixes in CSS to display video properly.
Before http://i.imgur.com/9h7yz6T.png
After http://i.imgur.com/9fVzNng.png

3. Fix issues with `slick` gallery in post previews. Caused by a missing slick library on the page.

### CODE REVIEW

**Review by veidenbaums — CHANGES_REQUESTED** [review]
Looks like the are some issues with the content. 

To replicate:
1. Create a post
2. Update it to video type
3. Upload a video and add it in content
4. After the video add some content, save it and now search it

https://i.imgur.com/kzxI6ee.png, as you see the content, is under the video, but it should be under title ( the same happens also with embed etc. )

File: template-parts/content-video.php [hunk 1]
```diff
@@ -21,7 +21,10 @@
 
 		<?php the_title( sprintf( '<h2 class="entry-title"><a href="%s" rel="bookmark">', esc_url( get_permalink() ) ), '</a></h2>' ); ?>
 
-		<?php echo esc_html( get_the_content( '', false ) ); ?>
```
Comment (veidenbaums) [inline template-parts/content-video.php:24]: Need to adjust this, as by default if there is no excerpt it should take the content as an excerpt.

Comment (dmytro-kovalov) [reply template-parts/content-video.php:24]: That's exactly why the original issue has happened. In conjunction with `esc_html` it outputs the HTML markup. See
http://i.imgur.com/BvBkPJk.png
http://i.imgur.com/eiG5LvW.png

Comment (veidenbaums) [reply template-parts/content-video.php:24]: Then maybe lets use strip_tags ? 
The idea is that if users didn't add an excerpt, it should be generated from the content.  At the moment t didn't work for me like that. 🤔

Comment (dmytro-kovalov) [reply template-parts/content-video.php:24]: Looks like `the_excerpt` works more intelligently in this case. It extracts the part of the content if an excerpt is not provided. See https://imgur.com/Ir0L5Tf.png

**Review by veidenbaums — APPROVED** [review]

**Pull request merged**